src/components/video-editor/AnnotationOverlay.tsx

```
@@ -111,6 +113,22 @@ export function AnnotationOverlay({
           </div>
         );
 
+      case "blur":
+        const previewScaleFactor = containerWidth / BASE_PREVIEW_WIDTH;
```

Biome (2.4.10)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)